mlir/lib/Dialect/Rock/Transforms/AffixTuningParameters.cpp

```diff
@@ -58,6 +58,13 @@ void AffixTuningParameters::runOnOperation() {
 
   func.walk(
       [&](RockGemmWrapperInterface op) { affixTuningParametersImpl(op); });
+  func.walk([&](ReduceOp op) {
+    func::FuncOp funcOp = getOperation();
```

Comment:
This `funcOp` is the same as `func` in the captured scope.

Reply:
ah yeah! silly me. removed it.